api/valor_api/logging.py

```diff
@@ -34,15 +34,32 @@ async def handle_request_validation_exception(
     return response
 
 
+async def handle_http_exception(
+    request: Request, exc: HTTPException
+) -> Union[JSONResponse, Response]:
+    if exc.status_code >= 500:
+        logger.error(
+            "Valor HTTP exception",
+            method=request.method,
+            path=request.url.path,
+            hostname=request.url.hostname,
+            exc_info=exc,
+        )
+    return JSONResponse(
+        content={"status": exc.status_code, "detail": exc.detail},
+        status_code=exc.status_code,
+    )
+
+
 async def handle_unhandled_exception(
-    request: Request, exc: RequestValidationError
+    request: Request, exc: Exception
 ) -> Union[JSONResponse, Response]:
     logger.error(
         "Valor unhandled exception",
         method=request.method,
         path=request.url.path,
         hostname=request.url.hostname,
-        exception=str(exc),
+        exc_info=exc,
```

Comment:
Doing the same structlog exception logging for unhandled exceptions.  Previously we just showed a string representation of the stack trace.